src/argilla/utils/dependency.py

```diff
@@ -63,22 +63,29 @@ def _compare_versions(
         )
 
 
-def require_version(requirement: str, func_name: Optional[str] = None) -> None:
+def require_version(requirement: Union[str, List[str]], func_name: Optional[str] = None) -> None:
     """
     Perform a runtime check of the dependency versions, using the exact same syntax used by pip.
     The installed module version comes from the *site-packages* dir via *importlib_metadata*.
 
     Args:
-        requirement (`str`): pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
+        requirement (`str` or `List[str]`): pip style definition, e.g.,  "tokenizers==0.9.4", "tqdm>=4.27", "numpy"
         func_name (`str`, *optional*): what suggestion to print in case of requirements not being met
 
     Example:
     ```python
     require_version("pandas>1.1.2")
     require_version("datasets>1.17.0", "from_datasets")
+    require_version(["pandas>1.1.2", "datasets>1.17.0"])
     ```
     """
 
+    if isinstance(requirement, list):
+        if len(requirement) == 0:
+            return
```

Comment:
I think we can raise a `ValueError` exception here

Reply:
The bare return is intended at this point, it was just to stop calling the function when the list was empty, and given the function shouldn't return anything when the dependencies are met, I thought it made sense like this. It's implemented like in the following snippet from the same function, but maybe I should the the list treatment?

```python
# special case
if package == "python":
    got_version = ".".join([str(x) for x in sys.version_info[:3]])
    for op, want_version in wanted.items():
        _compare_versions(op, got_version, want_version, requirement, package, func_name=func_name)
    return
```

Reply:
I think the `ValueError` makes sense

Reply:
Done!